Cargo.toml

```diff
@@ -3,18 +3,22 @@ members = [
     "atrium-api",
     "atrium-common",
     "atrium-crypto",
+    "atrium-repo",
     "atrium-xrpc",
     "atrium-xrpc-client",
     "atrium-oauth/identity",
     "atrium-oauth/oauth-client",
     "bsky-cli",
     "bsky-sdk",
+
+    # "examples/concurrent",
+    "examples/firehose",
 ]
 # Examples show how to use the latest published crates, not the workspace state.
 exclude = [
-    "examples/concurrent",
-    "examples/firehose",
-    "examples/video",
+    # "examples/concurrent",
```

Comment:
Is there a reason why these are no longer excluded?

Reply:
Ah - I had forgotten about this.
I changed this because the examples were written against the last released version of `atrium` rather than the version present in the repository.
I needed to update the firehose example to incorporate the changes I have made here, and chose to change its dependencies to point at the crates in this repository.

The decision to have examples demonstrate the last public release seems to be an unusual one. I haven't ever seen this be done for any other Rust library.
Any thoughts about changing this?

Reply:
Indeed. The current policy of using the latest release version does not allow us to use newly added packages.

We've discussed this before and made it the way it is now,
https://github.com/sugyan/atrium/pull/100#issuecomment-1947725642

But maybe we should reconsider.

Reply:
Indeed - I think it would be wise to reconsider :)

One potential alternative to allow people to find examples for the previously released version would be to create git tags upon release and instruct users to look for the corresponding tag.
It actually looks like `release-plz` is already creating these tags for you (though appears to be doing so per-crate).